src/Nethermind/Directory.Packages.props

```diff
@@ -13,6 +13,7 @@
     <PackageVersion Include="BouncyCastle.Cryptography" Version="2.5.0" />
     <PackageVersion Include="Ckzg.Bindings" Version="2.0.1.1258" />
     <PackageVersion Include="Colorful.Console" Version="1.2.15" />
+    <PackageVersion Include="CommandLineParser" Version="2.9.1" />
```

Comment:
This is redundant. We already use the same CLI parser for everything else.

Reply:
Removed